04.ls/ls.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+COLUMN = 3
+SPACE = 1
+
+def main
+  current_path = '../../dummy/' # "#{Dir.getwd}/"
+  current_entries = Dir.entries(current_path).sort
+  hidden_removed_entries = current_entries.delete_if { |entry| entry.start_with?('.') }
+  distinct_entries = distinguish_entries(hidden_removed_entries, current_path)
+  row = (distinct_entries.length.to_f / COLUMN).ceil
```

Comment:
```suggestion
  row_count = (distinct_entries.length.to_f / COLUMN).ceil
  # もしくは row_size など
```

とした方が数値であることが伝わりやすいと思います。
（`row`だと行データが含まれているようにも見える）

`COLUMN`にも同じことが言えますね。